flashinfer/cute_dsl/sparse/sm100_blk64/dispatch_helpers.py

```
@@ -0,0 +1,692 @@
+    """Avoid optional generated dialect imports that are broken in this environment."""
+    import importlib
+    import sys
+    import types
+
+    for suffix in ("arith", "dialect_proxy", "gpu", "lru_cache_ir", "op"):
+        canonical = f"cutlass._mlir_helpers.{suffix}"
+        alias = f"cutlass.base_dsl._mlir_helpers.{suffix}"
+        try:
+            sys.modules.setdefault(alias, importlib.import_module(canonical))
+        except Exception:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)